content/strategy-goals/strategy/use-cases/dev-onboarding.md

```diff
@@ -17,7 +17,7 @@ If we unpack this a bit further, understanding relates to:
 1. Knowing who is responsible for or has contextual knowledge about the code
 1. Knowing who updated the code and why
 
-This kind of understanding is central to both onboarding and velocity, and research shows that revenue growth among companies who invest in the velocity of their developers see revenue growth upto 4-5 times faster than those who don’t.
+This kind of understanding is central to onboarding.
```

Comment:
Moving this research to the higher-level page: https://github.com/sourcegraph/handbook/pull/2625/files